WcaOnRails/app/views/doorkeeper/authorized_applications/index.html.erb

```diff
@@ -1,5 +1,16 @@
-<header class="page-header">
-  <h1><%= t('doorkeeper.authorized_applications.index.title') %></h1>
+<% provide(:title, t('doorkeeper.authorized_applications.index.title')) %>
+<%
+  # FIXME panic button, I removed the navbar.
+  # For some reason the locale is correctly set in the params,
+  # but the default_url_options used in the navigation doesn't have it...
```

Comment:
The bug is that at some point when including the navbar, this [line](https://github.com/viroulep/worldcubeassociation.org/blob/i18n/WcaOnRails/app/helpers/application_helper.rb#L118) was called, and the application would crash because for some reason no locale is present in the `default_url_options`. Adding the `id:` key in front of the competition id makes the application generate a non-localized url.

Maybe this issue will disappear when rebasing though, I could see that the file has changed a lot !